### PR TITLE
Add admin deployment instructions for Mac

### PR DESCRIPTION
**Pull request by lenacohen**
Add the instructions and plist file I used to apply managed storage settings to Privacy Badger locally.

### CODE REVIEW

File: doc/configuration.plist [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+  <key>com.google.Chrome.extensions.cfoidkdkbbpjjlaidafdgkeeglomoaij</key>
```
Comment (ghostwords) [inline doc/configuration.plist:5]: Let's use the production ID (`pkehgijcmpdhfbdbbnkijodmdjhbjlgp`) as we do in the other samples.

File: doc/admin-deployment.md [hunk 2]
```diff
@@ -63,6 +63,10 @@ This policy also prevents the new user welcome page from launching upon Privacy
 }
 ```
 
+### Chrome on Mac
+
+Follow instructions [here](https://www.chromium.org/administrators/configuring-policy-for-extensions/#mac) to add an extension policy via plist file. Depending on your shell environment, you may need to change the syntax of some commands. Use [this plist file](/doc/configuration.plist) as a template, subtituting the extension ID for the ID of your locally installed Privacy Badger. Add keys in the format of `showCounter` to apply additional settings.
```
Comment (ghostwords) [inline doc/admin-deployment.md:68]: >Depending on your shell environment, you may need to change the syntax of some commands.

Should probably explain this with an example. Or remove entirely.

> Add keys in the format of `showCounter` to apply additional settings.

Can probably remove this line, or replace it with something like "The following policy disables the tracker count badge on Privacy Badger's icon."

Comment (ghostwords) [reply doc/admin-deployment.md:68]: Should we move all the policy examples into a subdirectory? Maybe `doc/sample-admin-policies`?

**Review by ghostwords — COMMENTED** [review]
Thank you!

File: doc/admin-deployment.md (hunk 2 above)
Comment (lenacohen) [reply doc/admin-deployment.md]: Good call, I removed the unnecessary instructions and moved the 3 policy examples into a new folder

**Pull request merged**